tests/abi/tools/revng-abi-verify/Verify.cpp

```diff
@@ -0,0 +1,381 @@
+/// \file Verify.cpp
```

Comment:
Drop in favor of `revng model opt -verify`. Eventually `revng model opt` will verify by default.